tests/test_dataset.py

```diff
@@ -1166,6 +1169,21 @@ def test_serialize_deserialize(datasets):
             assert arr_have == pytest.approx(arr_expected)
 
 
+def n_points(dataset):
+    # used in multiprocessing test
+    return dataset.n_points
+
+
```

Comment:
Can we stick this inside the test as a nested function, or just use a lambda _in situ_? In library code I'd be tempted to use `operator.attrgetter`, but probably not worth it in a test.

Reply:
> Can we stick this inside the test as a nested function

I don't think we can. multiprocessing will complain as it needs globals, not locals

Reply:
Ah, good shout. Too bad.